ooniapi/services/reverseproxy/scripts/docker-entrypoint.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+# vim:sw=4:ts=4:et
+
+set -e
+
+# Copied from: 
+# https://github.com/nginxinc/docker-nginx/blob/cffeb933620093bc0c08c0b28c3d5cbaec79d729/mainline/debian/docker-entrypoint.sh
+
+entrypoint_log() {
+    if [ -z "${NGINX_ENTRYPOINT_QUIET_LOGS:-}" ]; then
+        echo "$@"
+    fi
+}
+
+if [ "$1" = "/usr/bin/supervisord" ] || [ "$1" = "nginx-debug" ]; then
```

Comment:
Should you maybe also remove the `[ "$1" = "nginx-debug" ]` since you removed the `[ "$1" = "nginx" ]` line? Or maybe just add the supervisord to the check as an extra `||`?

Reply:
Yeah I changed `nginx-debug` to `supervisord` so that it handles both calling styles